tellor_all_languages/test/test_build_status_fail.js

```diff
@@ -0,0 +1,144 @@
+const BuildStatusCheck = artifacts.require("./BuildStatusCheck.sol");
+const Tellor = artifacts.require("TellorPlayground.sol");
+
+//Helper function that submits and value and returns a timestamp for easy retrieval
+const submitTellorValue = async (tellorOracle, requestId, amount) => {
+  //Get the amount of values for that timestamp
+  let count = await tellorOracle.getNewValueCountbyRequestId();
+  await tellorOracle.submitValue(requestId, amount);
+  let time = await getTimestampbyRequestIDandIndex(requestId, count.toString());
+  return time.toNumber();
+};
+
+contract("UsingTellor Tests", function (accounts) {
+	let buildStatusCheck;
+	let tellorOracle;
+
+	beforeEach("Setup contract for each test", async function () {
+	tellorOracle = await Tellor.new();
+	buildStatusCheck = await BuildStatusCheck.new(tellorOracle.address);
+	});
+
+	it("Update Price", async function () {
+	
+		// Specify which data (type) is requested from the oracle. (Should become something like 59, a new entry for TruCol)
+		const requestId = 1; 
+
+
+	// -----------------------------------------Helper Functions ----------------------------
+	// Encode a string to a number 
+	// Source: https://stackoverflow.com/questions/14346829/is-there-a-way-to-convert-a-string-to-a-base-10-number-for-encryption
+	function encode(string) {
+		var number = "0x";
+		var length = string.length;
+		for (var i = 0; i < length; i++)
+			number += string.charCodeAt(i).toString(16);
+		return number;
+	}    	
+    	
+    // Function that runs some incoming shell command (not bash)
+	const exec = require('child_process').exec;
+	function os_func() {
+		this.execCommand = function (cmd) {
```

Comment:
Not sure what `this` is supposed to be here. It seems like `os_func` is an async function (or at least that would be more standard for promises)
.